pkgs/applications/virtualization/qemu/default.nix

```diff
@@ -274,6 +265,7 @@ stdenv.mkDerivation (finalAttrs: {
     license = licenses.gpl2Plus;
     maintainers = with maintainers; [ eelco qyliss ];
     platforms = platforms.unix;
+    broken = stdenv.hostPlatform.system == "x86_64-darwin";
```

Comment:
could you leave a comment here letting others know it was marked "broken" out of caution and lack of hardware to use for testing when it was updated?

Reply:
It's not marked broken "out of caution", it's marked broken because it doesn't build.

Reply:
sorry, I misread the conversation, you're right.